pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/Utils.groovy

```diff
@@ -351,4 +352,29 @@ public class Utils {
         return knownTypes
     }
 
+    /**
+     * Determines whether a given {@link UninstantiatedDescribable} is of a given type.
+     *
+     * @param ud The {@link UninstantiatedDescribable} to check
+     * @param base The {@link Class}
+     * @return True if the uninstantiated describable is of the type given
+     */
+    public static boolean isOfType(UninstantiatedDescribable ud, Class<?> base) {
+        Descriptor d = SymbolLookup.get().findDescriptor(base, ud.symbol)
+        return d != null
+    }
+
+    /**
+     * @param c The closure to wrap.
+     */
+    public static StepsBlock createStepsBlock(c) {
```

Comment:
A `Closure` - I feel like there was a reason I didn't specify the type originally, but I'll try it again with it typed.